Dockerfile

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 #
 # Dockerfile to build PowerMax CSI Driver
-FROM docker.io/centos:centos8.3.2011
+FROM rockylinux:9
```

Comment:
Why is rocklinux:9 being used? Is this appropriate to use and is that what we are doing in other areas of CSM?

Reply:
Yes @gallacher, have been replicating what is being used in PowerScale

Reply:
@shanmydell , to be consistent across our repositories can we use the build.sh script in this directory to build a base image using UBI micro? Can the Makefile invoke this script instead of relying on a hardcoded base image in the Dockerfile?

Reply:
@tdawe: We are using the build.sh script to build the images that are getting posted to DockerHub, this command is for quick image building, should I add another command to build completely, this has been the behavior since Day 1. Also, PowerScale has having same code as the one here.